chihacknight/il-solar-map
# Planned fix

## Overview

* updates the latest data
* adds a table and chart of planned projects (related to #57 
* closes #60. many planned projects were being double counted, so our total planned is now 4,747

### Demo

New table and chart on about page:
![Screenshot 2024-07-05 at 2 26 39 PM](https://github.com/chihacknight/il-solar-map/assets/919583/cace5ecb-e22d-4472-bfa4-0b923c669e66)

Planned map accurately showing fewer projects:
![Screenshot 2024-07-05 at 2 27 11 PM](https://github.com/chihacknight/il-solar-map/assets/919583/2001fe4e-2985-45fa-9f79-8b00b11dc170)


## Testing Instructions

* view the about page and confirm the new planned table and chart shows up
* map should show far fewer planned Small and Large DG projects
